druid-shell/src/platform/gtk/window.rs

```diff
@@ -415,6 +429,18 @@ impl WindowHandle {
         }
     }
 
+    pub fn set_resizable(&self, resizable: bool) {
```

Comment:
In general we expect the various platform `WindowHandle` and other types to be used through the shared `WindowHandle` struct in `druid-shell/src/window.rs`, and so I would add these methods there, wrapping the method on each platform type; this will require you to add stubs of these methods to the other platforms.

Reply:
Uh, I already added the stubs for other platforms, so it should be enough to add the two methods to the WindowHandle right?

Reply:
I saw you had the methods for builder, but I didn't see them for WindowHandle? I might have missed it. :)

Reply:
Yep you're right, I missed the WindowHandle, adding it now